README.md

```diff
@@ -1,34 +1,65 @@
-# Crypto Market
+# 📖 Crypto Market
 
 Cryptic World is a Project that uses a Crypto API, to fetch data about Crypto Currencies. You can see all the basic information about the coins on the home page and you can filter them to find a specific coin. You can also click on a coin to find more information about that specific coin.
 
+# 🛠 Built With
+
+React
+Redux
+CSS
+
 # Live demo
```

Comment:
- [ ] Kindly update your README with the live demo link of your project and not a loom video presentation link.